Solutions/BitSight/Analytic Rules/BitSightBadOpenPortsDetected.yaml

```diff
@@ -0,0 +1,47 @@
+id: f65bfcdd-4da8-41fd-bca1-de6599400cd7
+name: BitSight - bad open ports detected
+description: |
+  'Rule helps to detect whenever there is a new bad open ports found in BitSight.'
+severity: Low
+status: Available
+requiredDataConnectors:
+  - connectorId: BitSight
+    dataTypes:
+      - BitSightFindingsData
+queryFrequency: 1d
+queryPeriod: 24h
+triggerOperator: GreaterThan
+triggerThreshold: 0
+tactics:
+  - Discovery
+query: |
+    let timeframe = 24h;
+    BitSightFindingsData
+    | where ingestion_time() > ago(timeframe)
+    | where RiskVector == "open_ports"
+    | extend Severity = case( toreal(Severity) <= 3.9 and toreal(Severity) >= 0.0, "Informational", 
+                              toreal(Severity) <= 6.9 and toreal(Severity) >= 4.0, "Low",
+                              toreal(Severity) <= 8.9 and toreal(Severity) >= 7.0, "Medium",
+                              toreal(Severity) <= 10.0 and toreal(Severity) >= 9.0, "High",
+                              "Informational")
+    | project FirstSeen, CompanyName, Severity, RiskVector, TemporaryId, RiskCategory
+incidentConfiguration:
+    createIncident: true
+    groupingConfiguration:
+      enabled: false
```

Comment:
groupingConfiguration is set to false here. If not required, please remove from the YAML.

Reply:
@devikamehra We have removed the above grouping block as it was set to true and there is no meaning of that. Thanks